torch/csrc/autograd/functions/batch_normalization.cpp

```diff
@@ -206,7 +206,7 @@ auto BatchNormBackward::apply(const variable_list& grad_outputs) -> variable_lis
                                  std::move(grad_bias));
   return wrap_outputs(all_inputs, std::move(outputs), [&](FunctionFlags f) {
     return std::make_shared<BatchNormBackwardBackward>(
-      f, *this, std::move(save_mean), std::move(save_std),
```

Comment:
It seems that they are not used anywhere in BackwardBackward so there's no point in saving them. A side effect of this move was that save_mean and save_std were set to NULL in Backward, so it segfaulted if you tried to use it again